views/lib/view.js

```diff
@@ -114,7 +114,7 @@ export default class View {
    * @return {LanguageDataset}
    */
   static get dataset () {
-    return LDF.getDataset(this.languageID)
+    return LDF.getDatasets(this.languageID)[0]
```

Comment:
Maybe it's better to add a comment explaining why the first dataset only is returned and what this dataset is?

Reply:
Added